Core/include/Acts/Vertexing/IterativeVertexFinder.hpp

```diff
@@ -75,9 +75,9 @@ class IterativeVertexFinder {
     /// @param lin Track linearizer
     /// @param sfinder The seed finder
     /// @param est ImpactPointEstimator
-    Config(const vfitter_t& fitter, Linearizer_t lin, sfinder_t sfinder,
+    Config(vfitter_t fitter, Linearizer_t lin, sfinder_t sfinder,
            const IPEstimator& est)
```

Comment:
```suggestion
           IPEstimator est)
```

I would make the copy explicit